react/components/Table/util/index.js

```diff
@@ -0,0 +1,17 @@
+import actionTypes from './actionTypes'
```

Comment:
What I said before, you could just put those functions here and export it, it seems simplier and easier do review and create less files

Reply:
Done